### ISSUE TITLE
请问有MASKDISTILL论文地址么

### ISSUE THREAD

**Issue by swjtulinxi** [open-mmlab/mmselfsup#780]
**Checklist**

请问有MASKDISTILL论文地址么，什么时候能集成这个模型代码呢

**Comment by drkostas** [comment]
I implemented MaskDistill from scratch in PyTorch, reproduced the paper's experiments, and open sourced the code and weights: https://github.com/drkostas/MaskDistill-PyTorch

I needed this for my research and the existing unofficial implementation (https://github.com/bwconrad/masked-distillation) was not giving me the expected results, so I ended up writing everything from scratch. ViT-Base pretrained for 300 epochs with CLIP teacher reproduces:

- Finetuning: 84.8% top-1 (paper reports 85.3%)
- Linear probe: 76.3% top-1
- Semantic segmentation: 52.6 mIoU (paper reports 53.8)
- k-NN: 75.6%
- Object detection: 44.4 bbox mAP
- Instance segmentation: 40.1 segm mAP

All checkpoints on HuggingFace: https://huggingface.co/drkostas/MaskDistill-ViT-Base
